tools/travis.ps1

```diff
@@ -201,9 +201,11 @@ elseif($Stage -eq 'Build')
     }
 
     $pesterParam = @{
-        'binDir'   = $output
-        'PassThru' = $true
-        'Terse'    = $true
+        'binDir'     = $output
+        'PassThru'   = $true
+        'Terse'      = $true
+        'Tag'        = @()
+        'ExcludeTag' = @('RequireSudoOnUnix')
```

Comment:
you need to specify two different values for '-OutputFile' and update https://github.com/PowerShell/PowerShell/blob/master/.travis.yml#L19 to upload both files

Reply:
Do we want to store the test results in output files as in ./tools/appveyor.psm1? Currently they are stored in variables ($pesterPassThruNoSudoObject, $pesterPassThruNSudoObject) and there was no -OutputFile specified for Start-PSPester cmdlet in travis.ps1

Reply:
OutputFile has a default.  travis.yml is assuming the default